### ISSUE TITLE
to-10.1-move-radius-configuration-parmeters.pl doesn't handle existing radius_configuration correctly

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5671]
**Describe the bug**
If you have changed some RADIUS configuration parameters before upgrading to 10.1.0 and you run `to-10.1-move-radius-configuration-parmeters.pl`, new files under `/usr/local/pf/conf/radiusd/ `are created with wrong values.

Parameters are:
- eap_authentication_types
- eap_fast_opaque_key
- eap_fast_authority_identity
- ocsp_enable 
- ocsp_override_cert_url 
- ocsp_url 
- ocsp_use_nonce 
- ocsp_timeout 
- ocsp_softfail 

**To Reproduce**
1. Set `eap_authentication_types=GTC,MD5,MSCHAPv2,PEAP,TLS,TTLS,PAP` in `pf.conf`
1. Run `to-10.1-move-radius-configuration-parmeters.pl`

```console
# cat /usr/local/pf/conf/radiusd/eap_profiles.conf
[default]
eap_authentication_types=1
```

**Expected behavior**
`/usr/local/pf/conf/radiusd/eap_profiles.conf` should contain old values of `pf.conf`

**Comment by nqb** [comment]
I fixed this issue in 4da38e9a6ac63e13bd26dbc9fb1165b707f5c565.

There was another issue with this script: if `ocsp_softfail` has been redefined in `pf.conf`, option was written to `/usr/local/pf/conf/radiusd/eap_profiles.conf` in place of `/usr/local/pf/conf/radiusd/ocsp.conf`. Fixed in 574d3fde59c21c345a467bb444b68dcc6f0effb2

**Comment by nqb** [comment]
I will backport this to maintenance but a patch release will be necessary.
